common/source/docs/common-ads-b-receiver.rst

```diff
@@ -154,7 +154,9 @@ When starting SITL use the following command:
 
 ::
 
-    sim_vehicle.sh -A --adsb --console --map
+    sim_vehicle.py --console --map
+
+At the SITL console set the ``SIM_ADSB_COUNT`` parameter to idnicate the number of nearby aircraft to simulate.

```

Comment:
I would suggest also mentioning the other options:
At the SITL console set the quantity of simulated aircraft to 5 with the command ``param set SIM_ADSB_COUNT 5`` . Values 0 - 100 are available. You can also set the altitude (SIM_ADSB_ALT) and radius (SIM_ADSB_RADIUS) of the simulated aircraft